election/serializers.py

```diff
@@ -89,9 +89,19 @@ def __init__(self, name, idx, profile, grade):
         self.profile = profile
         self.grade = grade
 
-
 class CandidateSerializer(serializers.Serializer):
     name = serializers.CharField()
     id = serializers.IntegerField(min_value=0)
     profile = serializers.DictField(child=serializers.IntegerField())
     grade = serializers.IntegerField(min_value=0, max_value=settings.MAX_NUM_GRADES)
+    num_votes = serializers.IntegerField(min_value=0)
+
+class LinkSerializer(serializers.Serializer):
+
+    id_election=serializers.CharField()
+    select_language = serializers.CharField(max_length=2,default="en")
+    creator_emails = serializers.ListField(

```

Comment:
It's really just a detail, but I would prefer to call it `emails` instead of `creator_emails`. `creator_emails` is quite opinionated: it assumes we are going to use all the time the endpoint as we intended first, whereas it could be use otherwise.